.travis.yml

```diff
@@ -76,7 +76,7 @@ install:
   - python setup.py install
 
 script:
-  - py.test xarray --cov=xarray --cov-report term-missing --verbose
+  - py.test xarray --cov=xarray --cov-report term-missing --verbose --skip-flakey
```

Comment:
Can you make `--skip-flakey` an environment variable so we can make one of our "allowed failure" matrix builds not skip flakey tests?